timeless/sync/sync.py

```diff
@@ -5,9 +5,12 @@
      database. After implementing this, remove @unittest.skip annotation from
      test_sync_location in test_sync.py. Do not forget to create cron script and
      put int into scripts folder to make this sync periodic.
-
+    @todo #24:30min Implement synchronization between Poster and Database for
+     Tables. Data coming from Poster has priority upon data stored in our
+     database. Do not forget to create cron script and
```

Comment:
@Vatavuk I think here should be a `celery task` instead of cron script.
@emilianodellacasa WDYT?

Reply:
@vladarefiev @emilianodellacasa I've changed the text `cron script` to `periodic script` . Maybe we can postpone the decision.